Selenium_Task80/TestProject2/LoginTests.cs

```diff
@@ -0,0 +1,53 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Selenium_Task80;
+using System.Threading;
+using System;
+
+namespace LoginTests
+{
+    public class Tests
+    {
+        public IWebDriver driver;
+
+        [SetUp]
+        public void BeginTest()
+        {
+            driver = new ChromeDriver();
+            driver.Url = "https://www.yandex.com";
+            Screenshot screenshot = ((ITakesScreenshot)driver).GetScreenshot();
+            screenshot.SaveAsFile("./Screenshot/screenshot.jpg", ScreenshotImageFormat.Jpeg);
```

Comment:
System.IO.DirectoryNotFoundException   Exception was thrown. Try to use GetCurrentDirectory and ccreate a separste folder for this purposes
 
 Also better to use new name for each  run. Example: TestName+DateTime